Networking/NetworkingTests/Remote/ProductVariationsRemoteTests.swift

```diff
@@ -117,6 +117,55 @@ final class ProductVariationsRemoteTests: XCTestCase {
         wait(for: [expectation], timeout: Constants.expectationTimeout)
     }
 
+    // MARK: - Load single product variation tests
+
+    /// Verifies that loadProductVariation properly parses the `product-variation` sample response.
+    ///
+    func test_load_single_ProductVariation_properly_returns_parsed_ProductVariation() throws {
```

Comment:
I think we should stop using "properly". I feel like it's implied. 😄

Reply:
hah yea, removed in https://github.com/woocommerce/woocommerce-ios/pull/2915/commits/26e5adeae232736e45eb0e54e0409e259a190c16